Cubical/HITs/Sn/Base.agda

```diff
@@ -0,0 +1,11 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.HITs.Sn.Base where
+
+open import Cubical.HITs.Susp
+open import Cubical.Data.Nat
+open import Cubical.Data.Empty
+open import Cubical.Foundations.Prelude
+
+S : ℕ → Type₀
+S zero = ⊥
+S (suc n) = Susp (S n)
```

Comment:
Can you please add equivalences with the other spheres in the library? (I think everything  you need should be in https://github.com/agda/cubical/blob/master/Cubical/HITs/Susp/Base.agda)